src/Workspaces/Core/Portable/Shared/Extensions/ICodeDefinitionFactoryExtensions.cs

```diff
@@ -24,6 +34,17 @@ internal static partial class ICodeDefinitionFactoryExtensions
                     SpecializedCollections.EmptyList<SyntaxNode>()));
         }
 
+        private static SyntaxNode CreateQualifiedThrowNotImplementStatement(
+            this SyntaxGenerator codeDefinitionFactory,
+            Compilation compilation)
+        {
+            return codeDefinitionFactory.ThrowStatement(
```

Comment:
That would affect more than just `NotImplementedException` though. We still want to add usings for parameters and return types.